types/ramda/ramda-tests.ts

```diff
@@ -1859,7 +1856,7 @@ class Rectangle {
 };
 
 () => {
-    const sortByNameCaseInsensitive = R.sortBy(R.compose<string, string, string>(R.toLower, R.prop("name")));
+    const sortByNameCaseInsensitive = R.sortBy(R.compose<Record<'name', string>, string, string>(R.toLower, R.prop("name")));
```

Comment:
The type parameters to `compose` were wrong here, which came to light as a result of these improved typings.